### PR TITLE
es6 style

### PR DESCRIPTION
**Pull request by Haidy777**

### CODE REVIEW

File: kraken.js [hunk 1]
```diff
@@ -1,85 +1,87 @@
...
-
-	// make sure to be backwards compatible
-	options = options || {};
-	if(typeof options === 'string') {
-		options = { otp: options };
-	}
+const request = require('requestretry');
+const crypto = require('crypto');
+const querystring = require('qs');
+const Pact = require('bluebird');
```
Comment (Mik13) [inline kraken.js]: can we rename that to bluebird (lowercase) again?

Comment (Haidy777) [reply kraken.js]: Sure, but I would suggest "Bluebird" since it gets called with new, if that's suitable for you?

Comment (Mik13) [reply kraken.js]: Yes of couse, you're right

**Comment by Haidy777** [comment]
@Mik13 It's changed now :)

**Comment by Mik13** [comment]
Thank you very much!

**Pull request merged**